content/paths/sign_requests__get_sign_requests.yml

```diff
@@ -0,0 +1,34 @@
+---
+operationId: get_sign_requests
```

Comment:
operationId and x-box-tags are currently updated and different in the Gundam schema as discussed on that PR, do we want to match these?

Reply:
We need to match the open api spec for the operationId and x-box-tags to the rest of the schema here because they're used to generate the documentation, they're less important in our repo because we don't use them to actually generate anything. From what I understood from the other repo we couldn't match this standard because we needed the SignRequest/ in front (https://git.dev.box.net/AppServices/public-api/pull/2#discussion_r328980)